src/main/java/rx/internal/operators/OperatorAll.java

```
@@ -61,7 +61,10 @@ public void onNext(T t) {
 
             @Override
             public void onError(Throwable e) {
-                child.onError(e);
+                if (!done) {
+                    done = true;
+                    child.onError(e);
+                }
```

Call `RxJavaHooks.onError` so no exception is lost.